datasources/packaging/postinst

```diff
@@ -1,7 +1,35 @@
 #!/bin/bash
 
-# create table for Rudder data sources plugin:
-# note that if you installed Rudder 4.1~alpha or 4.1~beta1, or a previous version
-# of the plugin, you can skip that step (which would fail with a "table exists" error)
-psql -U rudder -h localhost -d rudder -f /opt/rudder/share/plugins/datasources/datasources-schema.sql
+# Postinstall for Rudder datasource plugin
+RUDDER_OPT="/opt/rudder"
 
+SQL_CREDENTIALS=$(grep -c -E "^rudder.jdbc.(username|password)[ \t]*=" /opt/rudder/etc/rudder-web.properties || true)
```

Comment:
Should'nt we look for this password in /opt/rudder/etc/rudder-passwords.conf ?

Reply:
i've reused the way it's done in rudder-upgrade